ios/Plugin/Plugin.swift

```diff
@@ -283,19 +283,31 @@ public class AdMob: CAPPlugin, GADBannerViewDelegate, GADInterstitialDelegate, G
                 adUnitID = testingID
             }
 
-            self.interstitial = GADInterstitial(adUnitID: adUnitID)
-            self.interstitial.delegate = self
-            self.interstitial.load(self.GADRequestWithOption(call.getBool("npa") ?? false))
+            let request = self.GADRequestWithOption(call.getBool("npa") ?? false)
+            GADInterstitialAd.load(
+                withAdUnitID: adUnitID,
+                request: request,
+                completionHandler: { (ad, error) in
+                    if error != nil {
+                        NSLog("Received error on loading interstatial ad");
+                        call.error("Loading failed")
```

Comment:
I would attach the error to the response. So the app can capture UI logs of them.